### PR TITLE
docs: tells users where they are

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request

- Why submit this pull request?
* [ ] Bugfix
* [x] New feature provided
* [ ] Improve performance
* [ ] Backport patches
___
### Bugfix
- Description
tells users where they are in sidebar
- How to fix?
added a label of Apache Apisix to sidebar 
___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch
https://github.com/iamayushdas/apisix/tree/sidebar

- Related commits and pull requests
https://github.com/apache/apisix-website/issues/230

- Target branch
https://github.com/apache/apisix/tree/master

### CODE REVIEW

**Comment by iamayushdas** [comment]
@nic-chen some tests are failing can you look into this image

**Comment by iamayushdas** [comment]
@juzhiyuan have a look , in to this

**Comment by nic-chen** [comment]
@iamayushdas 
please have a look at TEST 9  in `t/admin/plugins.t`

```
#   Failed test 'TEST 9: confirm the plugin of auth type - response_body - like (repeated req 0, req 0)'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1586.
#                   '[{"name":"openid-connect","priority":2599},{"name":"wolf-rbac","priority":2555},{"name":"hmac-auth","priority":2530},{"name":"basic-auth","priority":2520},{"name":"jwt-auth","priority":2510},{"name":"key-auth","priority":2500},{"name":"authz-keycloak","priority":2000}]
# '
#     doesn't match '(?^:\[\{"name":"wolf-rbac","priority":2555\},\{"name":"hmac-auth","priority":2530\},\{"name":"basic-auth","priority":2520\},\{"name":"jwt-auth","priority":2510\},\{"name":"key-auth","priority":2500\}\])'
```

File: apisix/plugins/openid-connect.lua [hunk 1]
```diff
@@ -101,6 +101,7 @@ local schema = {
 local _M = {
     version = 0.1,
     priority = 2599,
+    type = 'auth',
```
Comment (juzhiyuan) [inline apisix/plugins/openid-connect.lua:104]: 🤔 This is not belonged to Auth plugin AFAIK, there

Comment (iamayushdas) [reply apisix/plugins/openid-connect.lua:104]: Oh yes yes, I forgot

**Pull request closed without merging**